tests/integration/replication-4.tcl

```diff
@@ -85,6 +85,42 @@ start_server {tags {"repl external:skip"}} {
     }
 }
 
+start_server {tags {"repl external:skip"}} {
+    start_server {} {
+        set master [srv -1 client]
+        set master_host [srv -1 host]
+        set master_port [srv -1 port]
+        set slave [srv 0 client]
+
+        test {First server should have role slave after SLAVEOF} {
+            $slave slaveof $master_host $master_port
+            wait_for_condition 50 100 {
+                [s 0 master_link_status] eq {up}
+            } else {
+                fail "Replication not started."
+            }
+        }
+
+        test {Replication of an expired key does not delete the expired key} {
+            $master debug set-active-expire 0
+            $master set k 1 ex 1
+            exec kill -SIGSTOP [srv 0 pid]
+            $master incr k
+            after 1000
+            exec kill -SIGCONT [srv 0 pid]
```

Comment:
let's add a comment here stating that this makes sure the INCR arrives to the replica after the key is logically expired